testing.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ ! -d $1/out ]; then
+	mkdir $1/out
+fi
+javac -d $1/out $1/src/Decide.java
```

Comment:
The line `javac -d $1/out $1/src/Decide.java` will cause issues, all files won't be named Decide.java

Reply:
Maybe this line is not needed at all? Try testing without it